lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_GeneratePercentilesFromProbabilities.py

```diff
@@ -127,6 +127,42 @@ def test_endpoints_of_distribution_exceeded(self):
             plugin._add_bounds_to_thresholds_and_probabilities(
                 threshold_points, probabilities_for_cdf, bounds_pairing)
 
+    @ManageWarnings(record=True)
+    def test_endpoints_of_distribution_exceeded_warning(
+            self, warning_list=None):
+        """
+        Test that the plugin raises a warning message when the constant
```

Comment:
We usually format unit test docstrings as:
```
"""Test that the plugin ...
```
This is because of how Sphinx formats these docstrings.